packages/dart/sshnoports/bin/npt.dart

```diff
@@ -112,6 +112,17 @@ void main(List<String> args) async {
         help: 'The remote host required'
             ' Alias: --rh',
       );
+      parser.addOption(
+        'local-host',
+        aliases: ['lh'],
+        help:
+            'Local IP address to bind to, or comma-separated list with fallbacks.'
+            ' When specified, npt will act as a gateway by binding to the'
+            ' first available IP instead of the default localhost (127.0.0.1).'
+            ' If multiple IPs are provided, only the first valid one is used.'
+            ' Example: --local-host 192.168.1.100,10.0.0.50'
+            ' Alias: --lh',
+      );
```

Comment:
Is there a reason we're not using the "defaultsTo" argument?

https://pub.dev/documentation/args/latest/args/ArgParser/addOption.html